app/src/main/java/com/example/playlistmaker/ui/settings/view_model/SettingsViewModel.kt

```diff
@@ -0,0 +1,62 @@
+package com.example.playlistmaker.ui.settings.view_model
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+
+import com.example.playlistmaker.creator.Creator
+import com.example.playlistmaker.domain.settings.SettingInteractor
+import com.example.playlistmaker.domain.sharing.SharingInteractor
+
+
+class SettingsViewModel(
+    private val sharingInteractor: SharingInteractor,
+    private val settingInteractor: SettingInteractor,
+) : ViewModel() {
+    private var currentThemeMutable = MutableLiveData<Boolean>()
+    val currentTheme: LiveData<Boolean> = currentThemeMutable
```

Comment:
В андроиде принято разделять названия таких штук вот таким образом (с помощью символа "_"):
```suggestion
    private var _currentTheme = MutableLiveData<Boolean>()
    val currentTheme: LiveData<Boolean> = currentTheme
```
Как пример в документации гугл - viewBinding (https://developer.android.com/topic/libraries/view-binding#fragments)

Reply:
Fixed